### PR TITLE
teardown test

### PR DESCRIPTION
**Pull request by maidh91**

### CODE REVIEW

File: test/02-teardown.test.js [hunk 1]
```diff
@@ -1,149 +1,149 @@
...
 const Helper = require('./helper')
-const harness = path.join(Helper.localDir, 'test', 'fixtures', 'harness')
+const teardownDir = path.join(Helper.localDir, 'test', 'fixtures', 'teardown')
+const teardownOsKillDir = path.join(Helper.localDir, 'test', 'fixtures', 'teardown-os-kill')
+const teardownExitCodeDir = path.join(Helper.localDir, 'test', 'fixtures', 'teardown-exit-code')
 
-test('teardown', async function ({ is, ok, plan, comment, teardown, timeout }) {
-  timeout(180000)
+test('teardown on pipe end', async function ({ ok, is, plan, comment, teardown, timeout }) {
+  if (isWindows) return
```
Comment (rafapaezbas) [inline test/02-teardown.test.js:13]: use `skip` test option like this -> https://github.com/holepunchto/localdrive/blob/3ccf20bf12290164ab5379684cc9d07476109e24/test/entry.js#L28

File: test/02-teardown.test.js [hunk 2]
```diff
@@ -1,149 +1,149 @@
...
 
-  const { code } = await running.until.exit
-  is(code, 0, 'exit code is 0')
+  const td = await teardownPromise
+  is(td, 'teardown', 'teardown executed')
 })
 
-test('teardown during teardown', async function ({ is, ok, plan, comment, teardown, timeout }) {
+test('teardown on os kill', async function ({ ok, is, plan, comment, teardown, timeout }) {
+  if (isWindows) return
```
Comment (rafapaezbas) [inline test/02-teardown.test.js:55]: same as above

File: test/02-teardown.test.js (hunk 1 above)
Comment (maidh91) [reply test/02-teardown.test.js]: updated

File: test/02-teardown.test.js (hunk 2 above)
Comment (maidh91) [reply test/02-teardown.test.js]: updated

**Comment by maidh91** [comment]
moved to https://github.com/holepunchto/pear/pull/432

**Pull request closed without merging**